app/lib/pages/map/page.tsx

```diff
@@ -115,11 +118,24 @@ const Page = () => {
               intro: "Done for the day? Make sure to logout!"
             }
           ],
+
+          initialStep: pageTourStep ? parseInt(pageTourStep, 10) : 0, //save tour progress
```

Comment:
Same as first comment this is also a component of the saving aspect of intro.js which isn't complete yet right?